clients/client-python/scripts/generate_version.py

```diff
@@ -37,7 +37,13 @@ def main():
         Version.COMPILE_DATE.value: compile_date,
     }
 
+    license_header = [
+        ";Copyright 2024 Datastrato Pvt Ltd.\n",
```

Comment:
Is it ok to use `;`?

Reply:
yeah, but seems `#` is more common, I changed it to `#` now